spacewalk/certs-tools/rhn_bootstrap.py

```diff
@@ -400,16 +359,12 @@ def processCommandline():
         options.http_proxy_password = ''
 
     # forcing numeric values
-    for opt in ['allow_config_actions', 'allow_remote_commands', 'force_bundle',
-        'no_bundle', 'no_gpg', 'no_up2date', 'traditional', 'up2date', 'verbose']:
+    for opt in ['force_bundle', 'no_bundle', 'no_gpg', 'verbose']:
         # operator.truth should return (0, 1) or (False, True) depending on
         # the version of python; passing any of those values through int()
         # will return an int
         val = int(operator.truth(getattr(options, opt)))
```

Comment:
I'm a bit surprised that `operator.truth` is needed here, I expect `optparse` to coerce the passed values into the defined types internally. It's unrelated to this PR, just took a closer at this code for the first time.

Reply:
I better keep it for now :-)